tests/installation/first_boot.pm

```diff
@@ -10,6 +10,12 @@ sub run() {
         return;
     }
 
+    # Wait for desktop to load after booting from an installed disk image
+    if ( check_var( "BOOT_HDD_IMAGE", 1 ) ) {
+        assert_screen "generic-desktop", 120;
+        return;
```

Comment:
hmm, that's broken. the 2nd argument is the default argument - so we would always go into that assert_screen even if not booting. Just get_var("BOOT_HDD_IMAGE") is what you want...

*But* I don't think this works this way. Because generic-desktop is not where you want to end your HDD_IMAGE is a text installation. 

I guess we need to come up with a formal way to express in the name of the HDD_IMAGES what to expect. But that sounds ugly too